test/e2e/kubectl/kubectl.go

```diff
@@ -1444,8 +1444,11 @@ metadata:
 			Expect(runOutput).To(ContainSubstring("abcd1234"))
 			Expect(runOutput).To(ContainSubstring("stdin closed"))
 
+			err := framework.WaitForJobGone(c, ns, jobName, wait.ForeverTestTimeout)
+			Expect(err).NotTo(HaveOccurred())
+
 			By("verifying the job " + jobName + " was deleted")
-			_, err := c.BatchV1().Jobs(ns).Get(jobName, metav1.GetOptions{})
+			_, err = c.BatchV1().Jobs(ns).Get(jobName, metav1.GetOptions{})
```

Comment:
So we can use `_, err = framework.WaitForJobGone(c, ns, jobName, 0)` here, right?

Reply:
Yeah, I guess we could, but I've just backported what we have in master right now.